### PR TITLE
[#13543] Add more pdo types

### PR DESCRIPTION
**Pull request by niden**
Hello!

* Type: bug fix | new feature | code quality | documentation
* Link to issue: #13151 #12223 #524 https://github.com/phalcon/cphalcon/pull/13225 (@zGaron) https://github.com/phalcon/cphalcon/pull/12523 (@Studentsov) https://github.com/phalcon/cphalcon/pull/12471 (@ruudboon)

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

This PR consolidates issues and PRs. Types supported for MySQL:

- `TYPE_BIGINTEGER`
- `TYPE_BIT`
- `TYPE_BLOB` 
- `TYPE_BOOLEAN` 
- `TYPE_CHAR` 
- `TYPE_DATE` 
- `TYPE_DATETIME` 
- `TYPE_DECIMAL` 
- `TYPE_DOUBLE` 
- `TYPE_ENUM` 
- `TYPE_FLOAT` 
- `TYPE_INTEGER` 
- `TYPE_JSON` 
- `TYPE_JSONB` 
- `TYPE_LONGBLOB` 
- `TYPE_LONGTEXT` 
- `TYPE_MEDIUMBLOB` 
- `TYPE_MEDIUMINTEGER` 
- `TYPE_MEDIUMTEXT` 
- `TYPE_SMALLINTEGER` 
- `TYPE_TEXT` 
- `TYPE_TIME` 
- `TYPE_TIMESTAMP` 
- `TYPE_TINYBLOB` 
- `TYPE_TINYINTEGER` 
- `TYPE_TINYTEXT` 
- `TYPE_VARCHAR` 

Thanks

### CODE REVIEW

**Comment by niden** [comment]
cc @sergeyklay Big one for you :)

**Comment by sergeyklay** [comment]
@niden Could you please rebase

File: boxfile.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+run.config:
```
Comment (sergeyklay) [inline boxfile.yml]: Could you send this file in a separate PR

**Comment by zGaron** [comment]
cool

**Review by sergeyklay — APPROVED** [review]
